compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -1707,6 +1707,7 @@ impl<'tcx> TypeErrCtxt<'_, 'tcx> {
                     ValuePairs::ExistentialProjection(_) => {
                         (false, Mismatch::Fixed("existential projection"))
                     }
+                    infer::DummyPair => (false, Mismatch::Fixed("values")),
```

Comment:
why that instead of `ValuePairs::DummyPair`? and why not `bug!()`

Reply:
oh lol i have no idea